src/Abp.AspNetCore/AspNetCore/Mvc/Proxying/AbpServiceProxiesController.cs

```diff
@@ -17,10 +17,10 @@ public AbpServiceProxiesController(IApiProxyScriptManager proxyScriptManager)
             _proxyScriptManager = proxyScriptManager;
         }
 
-        [Produces("text/javascript", "text/plain")]
```

Comment:
Is it really needed to remove Produces attribute (can we add it even return value has the mime type)?